src/login.coffee

```diff
@@ -169,42 +205,20 @@ define [
         last_name: $('input[name="new_last_name"]').val()
         email: $('input[name="new_email"]').val()
         email_confirmation: $('input[name="new_email_confirm"]').val()
-      $.ajax
-        type: "GET" # POST does not work in IE
-        data: user_data
-        datatype: 'json'
-        url:  "#{zutron_host}/zids/#{@my.zid}/email_change.json"
-        beforeSend: (xhr) ->
-          xhr.overrideMimeType "text/json"
-          xhr.setRequestHeader "Accept", "application/json"
-        success: (data) =>
-          if data? and data.errors # IE8 XDR Fallback
-            new ErrorHandler(data.errors, $form.parent().find(".errors"), 'changeEmailError').generateErrors()
-          else
-            @_setEmail(user_data.email)
-            events.trigger('event/changeEmailSuccess', data)
-            $('#zutron_account_form').prm_dialog_close()
-            @_triggerModal $("#zutron_success_form")
-        error: (errors) =>
-          new ErrorHandler($.parseJSON(errors.responseText), $form.parent().find(".errors"), 'changeEmailError').generateErrors()
+      onSuccess = ->
```

Comment:
I think this will need to change to `=>` so that `@_triggerModal` is found. I was getting an error in AG.